.github/PULL_REQUEST_TEMPLATE.md

```
@@ -1,3 +1,7 @@
-@coderabbitai summary
-@gemini-code-assist review
+
+
+<details>
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)